skills/dff_command_selector_skill/service_configs/dff-command-selector-skill/environment.yml

```diff
@@ -1,4 +1,5 @@
 SERVICE_PORT: 8148
 SERVICE_NAME: dff_command_selector_skill
 LANGUAGE: EN
-ROS_FSM_SERVER: http://robot-fake-server:8038
\ No newline at end of file
+ROS_FLASK_SERVER: http://ros-flask-server:6000
+FAKE: "False"
```

Comment:
why is it a string? to make it bool, you may use `FAKE: false`

Reply:
fixed